sign.js

```diff
@@ -219,6 +219,9 @@ var signAsync = module.exports.signAsync = function (opts) {
       var promise
       if (opts.identity) {
         debuglog('`identity` passed in arguments.')
+        if (opts.skipIdentityValidation) {
```

Comment:
I am wondering if it would be better to have this option `skipIdentityValidation` as `identity-validation` (as in the way we treat `gatekeeper-assess`) so the parameters are more consistent across `electron-osx-sign`?